MODULE.bazel

```diff
@@ -12,9 +12,15 @@ bazel_dep(name = "stardoc", version = "0.6.2", repo_name = "io_bazel_stardoc")
 bazel_dep(name = "rules_go", version = "0.39.1", repo_name = "io_bazel_rules_go")
 bazel_dep(name = "rules_oci", version = "1.6.0")
 
-# This is unfortunately requried by `rules_oci`.
+# This is unfortunately required by `rules_oci`.
 bazel_dep(name = "aspect_bazel_lib", version = "1.34.0")
 
+bazel_dep(
```

Comment:
What do you mean by necessary? It is a requirement for the the bzlmod tests to pass.

It's really nice to be able to run exactly what CI is running with buildifier installed here. I usually don't have buildifier installed on my system because I use these rules to be able to run it with the same consistent version and flags.

If you want me to delete this I can, I thought it was an improvement over what currently existed, but I'll leave it up to you.

Reply:
Oh, one thing worth noting. I set it as `dev_dependency = True` which means that downstream rules will not depend on this.

Reply:
Eh, while I am a fan of having things wired up with Bazel, I think buildifier is the one exception where I think it should be separate. That's how all the rules are configured in the https://github.com/bazelbuild org so would rather stick with that approach. If you feel strongly though, I'd definitely ask all dev tools be moved into some private package like `//tools/private` or something so any loads never accidentally an issue in the future.

Reply:
Ok, removed